web/pgadmin/static/js/helpers/ModalProvider.jsx

```diff
@@ -46,7 +46,7 @@ function renderExtraButtons(button) {
   case 'primary':
     return <PrimaryButton className='Alert-margin' startIcon={button.icon} onClick={button.onclick}>{button.label}</PrimaryButton>;
   case 'default':
-    return <DefaultButton className='Alert-margin' startIcon={button.icon} onClick={button.onclick}>{button.label}</DefaultButton>;
+    return <DefaultButton className='Alert-margin' startIcon={button.icon} onClick={button.onclick} color={button.color ?? 'default'}>{button.label}</DefaultButton>;
```

Comment:
why don't we just pass button.color as is and handle the default in DefaultButton.

Reply:
sure we can do that also.
will update